resources/session02/homework/http_server.py

```diff
@@ -31,11 +35,20 @@ def parse_request(request):
     if method != "GET":
         raise NotImplementedError("We only accept GET")
     return uri
+    
 
 
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    import mimetypes
+    mimetypes.guess_type('.html')
+    #uri = "/webroot/a_web_page.html"
+    try:
+        with open(uri, 'rb') as f:
+             return f       
```

Comment:
This is an open file handle (but once you return the block will exit, and it will be a closed file handle).  It's pretty useless in this form.  I think you mean to call `.read()` on it.